pom.xml

```diff
@@ -152,6 +152,8 @@
                                 <version>${errorprone.version}</version>
                             </path>
                         </annotationProcessorPaths>
+                    	<fork>true</fork>
+                    	<useIncrementalCompilation>true</useIncrementalCompilation>
```

Comment:
`true` appears to be the default value: https://maven.apache.org/plugins/maven-compiler-plugin/compile-mojo.html#useIncrementalCompilation

So, I don't think this change is doing anything.